feat: add /videos (#420)
layer5io/docs: content/en/videos/intermediate/meshery/meshery-teaser.md

---
title: "Meshery Playground Teaser Video"

> description: "Meshery Playground - figma for DevOps. Design, deploy and manage your cloudnative infrastructure."

Author: ```suggestion
description: "Meshery Playground - Explore features and capabilities of Meshery."
```
